pennylane/ops/functions/map_wires.py

```diff
@@ -100,17 +100,17 @@ def map_wires(
             return new_op
         return input.map_wires(wire_map=wire_map)
     if isinstance(input, (QuantumScript, QNode)) or callable(input):
-        return _map_wires_transform(input, wire_map=wire_map)
+        return _map_wires_transform(input, wire_map=wire_map, queue=queue)
```

Comment:
Why do we have this change? :thinking:

Reply:
This is for letting the `qml.map_wires` correctly use the `queue` argument to queue the operations correctly.